{"error":"\nInvalid `users.findFirst()` invocation in\n/mnt/c/Users/bhrya/Documents/3 ano/PI/Projeto/backend/src/services/AuthenticateUserService.ts:91:30\n\n   88 \n   89 static async login(data: AuthenticateUserLogin) {\n   90   const users = prisma.users;\nâ†’  91   const user = await users.findFirst(\n  The table `main.users` does not exist in the current database.","success":false}